Source/CesiumRuntime/Private/CesiumStaticMesh.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020-2021 CesiumGS, Inc. and Contributors
+
+#include "CesiumStaticMesh.h"
+
+UCesiumStaticMesh::UCesiumStaticMesh(
+    const FObjectInitializer& ObjectInitializer)
+    : UStaticMesh(ObjectInitializer) {}
+
+bool UCesiumStaticMesh::IsReadyForFinishDestroy() {
```

Comment:
Why is this needed, and why does it not need to call the base class implementation? Needs a comment at least.

Reply:
Was an experiment that seemed to reduce destruction cost in UE5, but no longer needed after the destruction changes that prevent main thread blocking. Forgot to delete these files, removed them now!